src/components/ContinueButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { LinkCSS, ImageCSS } from "../css";
+import { TableRowCell } from "../layout/Table";
+
+interface Props {
+    label: string;
+    linkTo: string;
+}
+
+const anchorStyles: any = {
+    backgroundColor: "#a1845c",
+    borderRadius: "20px",
+    color: "#ffffff",
+    display: "inline-block",
+    fontFamily: "'Guardian Text Sans',sans-serif",
+    fontSize: "17px",
+    lineHeight: "36px",
+    textAlign: "center",
+    textDecoration: "none",
+    minWidth: "200px",
+    padding: "0",
+    WebkitTextSizeAdjust: "none",
+    msoHide: "all"
+};
+
+const imgStyles: ImageCSS = {
+    verticalAlign: "middle",
+    border: "0"
+};
+
+export const ContinueButton: React.FC<Props> = ({ label, linkTo }) => {
+    const buttonMSOMarkup = `<!--[if mso]>
```

Comment:
I'm guessing this is taken directly from the existing email stuff, but it feels worth JSX-ing as much as possible here and keeping the raw text to a minimum. As working with JSX is much easier than raw strings for a lot of reasons.

Reply:
Yeah this is generated elsewhere and dropped into our component. I've JSX'd the markup as much as possible, so that all elements that are valid JSX have been used, but we still need a string to hold the outer HTML comment. It's still a messy component overall but this update is definitely an improvement.